### PR TITLE
Fix Perl POD commands.

### PR DESCRIPTION
**Pull request by drgrice1**
In some cases the lack of these empty lines will cause POD generation errors, and in other cases it just messes up the formatting of the generated POD.

Also, remove logs/standalone_results.log from git.  A .gitkeep file is added to ensure that the logs directory exists.  The renderer creates the log file as needed already.

### CODE REVIEW

**Comment by drgrice1** [comment]
It looks like you already have a branch that is doing this.  I didn't mean to step on your toes, I didn't see your branch until after I submitted this pull request.  I already had this branch though.

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,4 @@ node_modules/*
 *.pm.tdy
 *.bs
 .idea/
+logs/standalone_results.log
```
Comment (drdrew42) [inline .gitignore:17]: Line 10 already ignores `logs/*.log` -- I think that this logfile in particular was committed before the .gitignore was updated... we just need to `git rm` the file.

See drdrew42/renderer#75.

**Review by drdrew42 — CHANGES_REQUESTED** [review]
I'm looking to do some additional code cleanup on my branch before merging it -- if you want to go ahead and merge this, I'll deal with the conflicts. One way or another, these changes should be merged in by this evening. 😁

File: .gitignore (hunk 1 above)
Comment (drgrice1) [reply .gitignore]: Good point.   I missed that.

Comment (drgrice1) [reply .gitignore]: I removed that line.

**Comment by drgrice1** [comment]
I will merge this then.

**Pull request merged**